sonar-dotnet-shared-library/src/test/java/org/sonar/plugins/dotnet/tests/CoverageReportImportSensorTest.java

```diff
@@ -153,6 +155,10 @@ public void execute_coverage_no_main_file() throws IOException {
     context.fileSystem().add(new TestInputFileBuilder("foo", "Foo.cs").setLanguage("cs")
       .setType(Type.TEST).build());
 
+    File f = new File(fooPath);
+    assertThat(f.exists()).isTrue();
```

Comment:
the file does not get created on the Azure Pipelines VM and this is why the UT fails